packages/devtools/src/integrations/vscode.ts

```diff
@@ -144,7 +168,7 @@ export async function setup({ nuxt, options, openInEditorHooks, rpc }: NuxtDevto
       icon: 'bxl-visual-studio',
       category: 'modules',
       requireAuth: true,
-      view: !installed
+      view: !installed && !(vsOptions?.mode === 'tunnel')
         ? {
             type: 'launch',
             title: 'Install VS Code Server',
```

Comment:
Would be great if we could provide a page in the docs and change the guide here to mention that there are different options to install. Are you interested in helping that as well? Thanks!

Reply:
Yes I can do that!

Reply:
I'm assuming you mean updating [this](https://github.com/nuxt/devtools/blob/main/docs/content/1.guide/1.features.md#vs-code-server) page in the docs?

When you say guide, which guide are you referring to?

Reply:
Oh yeah, exactly!

Reply:
Should we make the ms-code-cli the default? It seems to be the recommended approach from MS, but it's different from the current behavior?

Reply:
Documentation updated: https://github.com/gsxdsm/devtools/blob/main/docs/content/1.guide/1.features.md#vs-code-server

Reply:
Note - would be good to publish antfu.vscode-server-controller into open-vsx marketplace so the install step works for code-server (from Coder) - https://open-vsx.org/user-settings/extensions